frappe/patches/v12_0/set_correct_assign_value_in_docs.py

```diff
@@ -1,25 +1,23 @@
 import frappe
 
+
 def execute():
-	frappe.reload_doc('desk', 'doctype', 'todo')
+	frappe.reload_doc("desk", "doctype", "todo")
+
+	ToDo = frappe.qb.Table("ToDo")
+	assignees = frappe.qb.GROUP_CONCAT("owner").distinct().as_("assignees")
```

Comment:
Custom functions should also be a part of the `frappe.qb.fn` namespace, given current design. Maybe we should separate this out though.

Let's have a common namespace for all function imports, perhaps `frappe.query_builder.functions` that will include all pypika inbuilts and our custom functions.